tests/backup/backup_partial_test.go

```diff
@@ -471,3 +474,323 @@ var _ = Describe("{PartialBackupSuccessWithPxVolumes}", func() {
+			err = Inst().Backup.BackupSchedulePolicy(schedulePolicyName, uuid.New(), BackupOrgID, schedulePolicyInfo)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of new schedule policy [%s]", schedulePolicyName))
+			schedulePolicyUid, err = Inst().Backup.GetSchedulePolicyUid(BackupOrgID, ctx, schedulePolicyName)
+			log.FailOnError(err, "Fetching uid of schedule policy [%s]", schedulePolicyName)
+			var wg sync.WaitGroup
+			for i := 1; i <= scheduledBackupCount; i++ {
+				scheduledBackupName := fmt.Sprintf("scheduled-backup-%v-%v", i, RandomString(6))
+				log.Infof("Creating scheduled backup [%s]", scheduledBackupName)
+				wg.Add(1)
+				go func(i int) {
+					defer wg.Done()
+					_, err = CreateScheduleBackupWithoutCheckWithVscMapping(scheduledBackupName, SourceClusterName, backupLocationName, backupLocationUid, appNamespaces, labelSelectors, BackupOrgID, "", "", "", "", schedulePolicyName, schedulePolicyUid, ctx, map[string]string{}, true)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup with schedule [%s]", schedulePolicyName))
+					scheduledBackups = append(scheduledBackups, scheduledBackupName)
+				}(i)
+				time.Sleep(timeBetweenScheduledBackups)
+			}
+			wg.Wait()
+		})
+
+		Step("Verify if the first scheduled backup is Success", func() {
+			log.InfoD("Verifying if the first scheduled backup is of Success status")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			var wg sync.WaitGroup
+			for _, schBkpName := range scheduledBackups {
+				wg.Add(1)
+				go func(schBkpName string) {
+					defer wg.Done()
+					firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, schBkpName, BackupOrgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching name of the first scheduled backup with schedule [%s]", schBkpName))
+					log.Infof("Validating if the first scheduled backup [%s] of the schedule [%s] is a success", firstScheduleBackupName, schBkpName)
+					err = BackupSuccessCheck(firstScheduleBackupName, BackupOrgID, MaxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second, ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying if the backup [%s] of schedule [%s] is a Success", firstScheduleBackupName, schBkpName))
+				}(schBkpName)
+			}
+			wg.Wait()
+		})
+
+		Step("Start a watcher to delete the data export CR of the first namespace to create a partial backup", func() {
+			log.InfoD("Starting a watcher to delete the data export CR of the first namespace to create a partial backup")
+			kdmpClient := kdmp.Instance()
+			watcher, err := kdmpClient.WatchDataExport(appNamespaces[0], metav1.ListOptions{})
+			if err != nil {
+				dash.VerifyFatal(err, nil, "Failed to start the watcher")
+				return
+			}
+			go func() {
+				defer func() {
+					watcher.Stop()
+					log.InfoD("Watcher stopped")
+				}()
+				ch := watcher.ResultChan()
+				oneHourTimeout := time.After(30 * time.Minute)
```

Comment:
Why are you calling it `oneHourTimeout`?
Also, it should timeout after 15-20 min right?